evmrpc/tracers_test.go

```diff
@@ -31,3 +31,29 @@ func TestTraceTransaction(t *testing.T) {
 		require.Greater(t, len(balance), 2)
 	}
 }
+
+func TestTraceBlockByNumber(t *testing.T) {
+	args := map[string]interface{}{}
+	args["tracer"] = "callTracer"
```

Comment:
can you also add a test for tracer=prestateTracer--same as what I did above?

Also for TraceBlockByHash